operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/DependentResourceManager.java

```diff
@@ -67,21 +67,15 @@ public void injectInto(EventSourceContext context) {
   public UpdateControl<R> reconcile(R resource, Context context) {
     initContextIfNeeded(resource, context);
 
-    dependents.stream()
-        .filter(dependent -> dependent.creatable() || dependent.updatable())
-        .forEach(dependent -> {
-          var dependentResource = dependent.getFor(resource, context);
-          if (dependent.creatable() && dependentResource == null) {
-            // we need to create the dependent
-            dependentResource = dependent.buildFor(resource, context);
-            createOrReplaceDependent(resource, context, dependent, dependentResource, "Creating");
-          } else if (dependent.updatable()) {
-            dependentResource = dependent.update(dependentResource, resource, context);
-            createOrReplaceDependent(resource, context, dependent, dependentResource, "Updating");
-          } else {
-            logOperationInfo(resource, dependent, dependentResource, "Ignoring");
-          }
-        });
+    dependents.stream().forEach(dependent -> {
```

Comment:
I would put this logic the the reconcile method of the resource, and just call here reconcile

Reply:
Probably makes sense, yes

Reply:
Done.